pkg/api/vlabs/types.go

```diff
@@ -226,32 +254,6 @@ type KeyVaultCertificate struct {
 	CertificateStore string `json:"certificateStore,omitempty"`
 }
 
-// OrchestratorType defines orchestrators supported by ACS
-type OrchestratorType string
```

Comment:
one small input it should be OcherstratorVersionHint, not Type. We should still have a short list of allowed Orchestrators.